Makefile

```
@@ -8,6 +8,7 @@ IMAGE_BASE ?= observability-operator
 
 VERSION ?= $(shell cat VERSION)
 OPERATOR_IMG = $(IMAGE_BASE):$(VERSION)
+OPERATOR_BUNDLE=observability-operator.v$(VERSION)
```

```suggestion
OPERATOR_BUNDLE=observability-operator-v$(VERSION)
```
Isn't this a more common way of adding a version to the name? I went looking for examples and found https://olm.operatorframework.io/docs/best-practices/channel-naming/ which also seems to suggest `name-version`. Might be less surprising for anyone who tries to parse this?

---

I am a bit confused about this comment. 
We aren't changing the existing channel names `stable, candidate, development` in this patch. 

If it is about the change proposed, that wouldn't work since `opm render <bundle-image>`  uses the format in PR. E.g. 

```
❯ opm render quay.io/rhobs/observability-operator-bundle:0.0.13 --output=yaml | ag '^name:'
name: observability-operator.v0.0.13
```

---

Got it. I just pointed it out cause the name format looked odd to me. But since its not under our control I'll merge this.